pulser-pasqal/pulser_pasqal/backends.py

```diff
@@ -268,6 +268,32 @@ class EmuMPSBackend(RemoteEmulatorBackend):
     _device_type = pasqal_cloud.DeviceTypeName.EMU_MPS
 
 
+class EmuSVBackend(RemoteEmulatorBackend):
+    """
+    Backend for executing quantum programs using the EMU-SV emulator.
+
+    The config supports various fields. For a complete list of accepted
+    parameters (passed as `**kwargs`), refer to the official EMU-SV documentation:
+    https://pasqal-io.github.io/emulators/latest/emu_sv/api/#svconfig
+
+    Args:
+        sequence: The quantum sequence to execute on the backend.
+        connection: An open PasqalCloud connection.
+        config: An EmulationConfig object to configure the backend. If not provided,
+            the default configuration will be used.
+        mimic_qpu: Whether to mimic the validations required for
+            execution on a QPU.
+    """
+
+    default_config = EmulationConfig(
+        observables=[BitStrings()],
+        gpu=True,
+        autosave_dt=float("inf"),
+        optimize_qubit_ordering=True,
+    )
```

Comment:
Did you check these values with the @pasqal-io/emulation-team ? I'm pretty sure some of them are specific to EMU MPS.
I'll let them chime in